scripts/lib/mkcloud-libvirt.sh

```diff
@@ -57,36 +57,85 @@ function libvirt_net_start()
         ip link set mtu 9000 dev $dev
     done
 
+    boot_mkcloud=/etc/init.d/boot.mkcloud
+    boot_mkcloud_d="$boot_mkcloud.d"
+    boot_mkcloud_d_cloud="$boot_mkcloud_d/$cloud"
+
     if [ -z "$NOSETUPPORTFORWARDING" ] ; then
+        # FIXME: hardcoded assumptions about admin net host range
```

Comment:
Huh? "I assume X but I would be surprised if my assumption was correct" - that doesn't make any sense ... If you don't expect X to be correct, then you aren't assuming it.

In any case, even if it is not currently possible for an `mkcloud` user to change the admin network host range from starting at `.81` to something else, the point is that this value should not be hardcoded here, and instead obtained from the single authoritative source of truth. Otherwise if the source of truth changes in the future, this will break.